spec/stack_master/parameter_resolvers/parameter_store_spec.rb

```diff
@@ -0,0 +1,50 @@
+RSpec.describe StackMaster::ParameterResolvers::ParameterStore do
+
+  describe '#resolve' do
+
+    let(:config) { double(base_dir: '/base') }
+    let(:stack_definition) { double(stack_name: 'mystack', region: 'us-east-1') }
```

Comment:
this should probably be an `instance_double`, which validates any methods called on it actually exist on an instance of the StackDefinition class

Reply:
*nod* Probably but dunno it's worth rejigging everything in this PR, it's done this way in:
- `parameter_resolvers/latest_ami_by_tags_spec`
- `parameter_resolvers/latest_ami_spec`
- `stack_master/parameter_resolvers/env_spec`
...etc
and I'd prefer to address this in a separate PR if we want to change the approach across a bunch of resolvers...